src/components/common/ParentFooter.js

```diff
@@ -37,6 +35,7 @@ export default function ParentFooter(props) {
             </Link>
           </div>
         </nav>
+        <div class="copyright">©{curYear} Story Squad HQ</div>
```

Comment:
Great job moving the copyright to under the links.

Reply:
Thanks Mitchell.